src/pages/guides/getting_started/code_playground_script_mode.md

```diff
@@ -0,0 +1,134 @@
+
+Script Mode is designed for rapid prototyping and learning. It provides a simplified environment where you can:
+
+- Test Document API calls directly
+- Learn how the Document APIs work
+- Experiment with document manipulation
+- Debug specific functionality
+
+**Note:** The code you write in this mode is equivalent to the code you would write and use in the `sandbox/code.js` file in an add-on project running locally.
+
+## When to Use Script Mode
+
+Use Script Mode when you want to:
+
+- Learn how the Document APIs work
+- Quickly experiment with Document API calls without UI considerations
+- Test specific API functionality
+- Debug isolated code snippets
+- Prototype document manipulation logic
+
+## How to Use Script Mode
+
+### Step 1: Select Script Mode
+
+1. Click the **Script** button in the top left corner of the playground window
+2. You'll see a single code editor where you can write your Document API code
+
+![Code Playground Script Mode](./img/script-mode.png)
+
+### Step 2: Write Your Code
+
+Enter your [Document API](../../references/document-sandbox/document-apis/index.md) code in the editor. You can:
+
+- Manipulate the document directly
+- Add shapes or text
+- Change styles and properties
+- Use the automatically available [`editor`](../../references/document-sandbox/document-apis/classes/Editor.md) object
+
+### Step 3: Execute Your Script
+
+Click the **Run Code** button in the right corner of the playground window to see changes in the current document.
+
+### Step 4: Configure Experimental APIs (Optional)
+
+If you want to use Document APIs that are currently marked experimental:
+
+1. Click on the properties icon to open the [Manifest JSON](../../references/manifest/index.md#requirements) editing modal
+2. Toggle **experimentalApis** to enable experimental features
+
+![Script Mode Manifest Settings](./img/manifest-props-script.png)
+
+## Key Features
+
+### Global Await Support
+
+The script runtime provides a global `async` wrapper, allowing you to use `await` directly when executing asynchronous code, without needing to wrap it in an `async` function:
+
+```js
+// The script runtime provides an async wrapper to allow this:
+const textNode = editor.context.selection[0];
+const lato = await fonts.fromPostscriptName("Lato-Light");
+```
+
+This is particularly useful for API calls that return promises, where an `await` is needed to pause the execution of an `async` function until the Promise is resolved or rejected.
+
+### Automatic Imports
+
+Script mode automatically imports the `express-document-sdk` modules, so you don't need to add import statements for the [Document APIs](../../references/document-sandbox/document-apis/index.md). However, if you do add import statements, it won't harm anything.
+
+### No UI Building
+
+Script mode is focused on Document API interactions and does not support building a user interface. If you want to create a UI, switch to [Add-on Mode](./code_playground_addon_mode.md).
```

Comment:
I feel calling out 'No UI Building' as a feature could be misleading. Can we move this paragraph under the 'What is Script Mode' section?